pymobiledevice3/cli/developer.py

```diff
@@ -155,6 +155,14 @@ def kill(service_provider: LockdownClient, pid):
         ProcessControl(dvt).kill(pid)
 
 
+@dvt.command('processIdForBundleId', cls=Command)
+@click.argument('appbundleidentifier')
+def processIdForBundleId(service_provider,appbundleidentifier) -> None:
```

Comment:
```suggestion
def process_id_for_bundle_id(service_provider: LockdownServiceProvider, app_bundle_identifier: str) -> None:
```
In general, the project is PEP8 and though not all project contains typing, any diff is expected to contain the correct typing.